go.mod

```diff
@@ -4,12 +4,13 @@ go 1.13
 
 require (
 	github.com/golang/mock v1.6.0
-	github.com/mattermost/mattermost-plugin-api v0.0.21
-	github.com/mattermost/mattermost-server/v6 v6.0.3
-	github.com/mholt/archiver/v3 v3.5.0
+	github.com/mattermost/mattermost-plugin-api v0.0.12-0.20200908143138-66edf222f7ea
+	github.com/mattermost/mattermost-server/v5 v5.25.1 // indirect
+	github.com/mattermost/mattermost-server/v6 v6.4.2
```

Comment:
Why this change?

Reply:
Reverted back to v5: https://github.com/mattermost/mattermost-plugin-gitlab/pull/280/commits/ad4533fb4bcd82272aed341bd922482382e2386a

Reply:
I think it is already using v6 .. this comment is from master branch mentioning v6: https://github.com/mattermost/mattermost-plugin-gitlab/blob/master/server/plugin.go#L305